packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -31,17 +31,75 @@ RED.palette.editor = (function() {
     var eventTimers = {};
     var activeFilter = "";
 
-    function semVerCompare(A,B) {
-        var aParts = A.split(".").map(function(m) { return parseInt(m);});
-        var bParts = B.split(".").map(function(m) { return parseInt(m);});
-        for (var i=0;i<3;i++) {
-            var j = aParts[i]-bParts[i];
-            if (j<0) { return -1 }
-            if (j>0) { return 1 }
+    var semverre = /^(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?(-(?<pre>[0-9A-Za-z-]+))?(\.(?<build>[0-9A-Za-z-.]+))?$/;
+	var NUMBERS_ONLY = /^\d+$/;
+
```

Comment:
The indenting appears off.

Reply:
I will fix the indentation, but what about the SemVer regex? The build server complains that this syntax is not correct in Node v8. It is valid in Node >= 10, but should this be corrected?